src/frontend/src/flows/dappsExplorer/teaser.ts

```diff
@@ -0,0 +1,85 @@
+import { html, TemplateResult } from "lit-html";
+import { dapps } from "./dapps";
+
+/** A teaser for the dapps explorer, acting as a button */
+export const dappsTeaser = ({
+  click,
+}: {
+  click: () => void;
+}): TemplateResult => {
+  return html`<article class="c-card c-card--narrow">
+    <h3 class="t-title t-title--discrete" role="presentation">
+      Dapps explorer
+    </h3>
+    <h2 class="t-title">Explore dapps</h2>
+    <button
+      class="c-click-area"
+      @click="${() => click()}"
+      aria-label="Show list of dapps"
+    >
+      <figure class="c-card__teaser c-marquee c-marquee--clickable">
+        ${marqueeList()}
+      </figure>
+    </button>
+  </article>`;
+};
+
+const marqueeList = (): TemplateResult => {
+  const itemsPerRow = 5;
+  const totalRows = 4;
+
+  const paddedDapps = repeatArray(dapps, itemsPerRow * totalRows);
```

Comment:
any particular reason why `repeatArray` and `chunkArray` do not take an object as parameter?

Reply:
If you don't mind, I would rather like. Maintance for the win.